Modules/xASL_module_ASL.m

```diff
@@ -755,18 +755,47 @@
     fprintf('\n%s', ['-> Detected ' xASL_num2str(nLists) ' list(s) for concatenating sessions']);
 end
 
+% Read and check a corresponding list with merging scalings
+if ~isfield(x.modules.asl, 'SessionMergingScaling')
+	% By default set as empty
+	x.modules.asl.SessionMergingScaling = {};
```

Comment:
Also comment here that it is a list of list, I thought it should be a [] but you are right, which I now only find out 2 lines further

Reply:
Good point. Added a comment.